src/util/mkcert/Makefile

```diff
@@ -22,15 +22,16 @@
 # Copyright (c) 2016, Oracle and/or its affiliates. All rights reserved.
 #
 
+include ../../Makefile.com
```

Comment:
It's probably more to do with the compiler version in omnios or the warnings that we enable, but I also had to do:

https://github.com/omniosorg/pkg5/pull/391/commits/bdd13600ce1d50e9b82ff26857db8ee8e50d245c

Reply:
I think you must enable `-Wextra` too, and I didn't.